content/docs/1.6.0/deploy/install/install-with-helm.md

```diff
@@ -8,9 +8,7 @@ In this section, you will learn how to install Longhorn with Helm.
 ### Prerequisites
 
 - Each node in the Kubernetes cluster where Longhorn will be installed must fulfill [these requirements.](../#installation-requirements)
-- Helm v2.0+ must be installed on your workstation.
-  - 1. Refer to the official documentation for help installing Helm.
-  - 2. If you're using a Helm version prior to version 3.0, you need to [install Tiller in your Kubernetes cluster with role-based access control (RBAC)](https://v2.helm.sh/docs/using_helm/#tiller-namespaces-and-rbac).
+- Helm v3 must be installed on your workstation.
```

Comment:
```
- Your workstation: Install [Helm](https://helm.sh/docs/) v3.0 or later.
```

Reply:
This is the exact wording in the "Install with Flux" and "Install with Fleet" files. I suggest we use the same here.